save-agent/src/nativeMain/kotlin/org/cqfn/save/agent/AgentConfiguration.kt

```diff
@@ -30,7 +30,7 @@ data class AgentConfiguration(
     val requestTimeoutMillis: Long,
     val executionDataRetryAttempts: Int,
     val executionDataInitialRetryMillis: Long,
-    val cliCommand: String = "./save-$SAVE_CORE_VERSION-linuxX64.kexe",
+    val cliCommand: String = "test -f run.sh && ./run.sh; ./save-$SAVE_CORE_VERSION-linuxX64.kexe",
```

Comment:
Are you sure it will work? It will execute `kexe` file too, but we expect the situation, when it's called from inside the script. So this will be more suitable:
```suggestion
    val cliCommand: String = "test -f run.sh && ./run.sh || ./save-$SAVE_CORE_VERSION-linuxX64.kexe",
```